CONTRIBUTING.md

```diff
@@ -81,6 +81,20 @@ We want your work to be readable by others; therefore, we encourage you to note
 
 - We encourage the use of Python [f-strings](https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python) where they make the code easier to read.
 
+- Please consider running [__psf/black__](https://github.com/python/black) on your Python file(s) before submitting your pull request.  This is not yet a requirement but it does make your code more readable and automatically aligns it with much of [PEP 8](https://www.python.org/dev/peps/pep-0008/). There are other code formatters (autopep8, yapf) but the __black__ formatter is now hosted by the Python Software Foundation. To use it,
+
+  ```bash
+  python3 -m pip install black  # only required the first time
+  black .
+  ```
+
```

Comment:
In my opinion, this part isn't necessary because when you create the pr the pre-commit bot automatically formats it because it's in the pre-commit